docs/project/contributing.md

```diff
@@ -130,3 +130,20 @@ Any user-facing change should be labeled with `include-changelog`.
 The PR title should contain a concise summary of the feature or fix and the description should have the GitHub issue number.
 When we publish a new version of lakeFS, we will add this to the relevant version section of the changelog.
 If the change should not be included in the changelog, label it with `exclude-changelog`.
+
+### User-Facing Changes Examples
+1. UI/UX modifications: Changes to the layout, color scheme, or navigation structure.
+1. New features: Adding functionality that users can directly interact with.
+1. Configuration changes: Updates to settings that users can adjust.
+1. Performance improvements: Enhancements that noticeably speed up the application.
+1. Bug fixes: Corrections to issues that users have encountered or reported.
+1. Security updates: Changes that address vulnerabilities or privacy concerns.
+
+### Non-User-Facing Changes:
+1. Code refactoring: Restructuring the codebase without changing its external behavior.
+1. Backend optimizations: Improvements to server-side processes that don't noticeably affect performance.
+1. Database schema changes: Modifications to the data structure that don't alter the user interface.
+1. Development tooling updates: Changes to build processes or development environments.
+1. Internal API modifications: Alterations to internal APIs that don't affect the public-facing functionality.
+1. Documentation updates: Changes to the project's documentation that don't impact the user experience.
+1. Imported libraries: Updates to third-party libraries that don't introduce new features, bug fixes, performance improvements or security updates.
```

Comment:
I don't think we need to add to the changelog third party updates at all regardless of security or otherwise. If it fixes a bug in lakeFS or adds performance it go into one of the former categories.